wealth-watch-script.sql

```diff
@@ -0,0 +1,35 @@
+create table user
+(
+id int unsigned not null unique auto_increment,
+name varchar(45) not null unique,
+password varchar(20) not null,
+primary key (id)
+);
+
+create table portfolio
+(
+id int unsigned not null unique auto_increment,
+user_id int unsigned not null,
+name varchar(50) not null unique,
+currency enum('CAD', 'USD', 'BRL') not null,
+primary key (id),
+constraint fk_portfolio_user foreign key (user_id) references user (id)
+);
+
+create table portfolio_item
+(
+id int unsigned not null unique auto_increment,
+portfolio_id int unsigned not null,
+symbol varchar(20) not null unique,
```

Comment:
This won't be unique because we can have multiple same symbols like the lots on yahoo finance.
![image](https://github.com/aleixoporpino/wealth-watch-api/assets/974517/54bc9d79-db32-47b2-a4d7-77decd853c8b)

Reply:
done.